app/models/comments.model.js

```diff
@@ -0,0 +1,19 @@
+const db = require('../../db/connection')
+
+exports.fetchCommentsByArticleId = (article_id) => {
+    let queryStr = `SELECT
+     * FROM 
+    comments 
+    WHERE comments.article_id = ${article_id}
+     ORDER BY created_at DESC`
+    return db.query(queryStr).then(({rows}) => {
+        if (rows.length === 0){
+        return Promise.reject(rows)
+        }
+        return rows
+    }).catch((err) => {
```

Comment:
No need for a second catch block here, the catch block in the controller should collect any error that occurs at any point in the promise chain